xbmc/pvr/windows/GUIWindowPVRGuide.h

```diff
@@ -80,7 +80,7 @@ namespace PVR
     std::atomic_bool m_bRefreshTimelineItems;
     std::atomic_bool m_bSyncRefreshTimelineItems;
 
-    CPVRChannelGroupPtr m_cachedChannelGroup;
+    std::shared_ptr<CPVRChannelGroup> m_cachedChannelGroup;
```

Comment:
forward decl for `CPVRChannelGroup` in this header missing. works by luck due to base class header include.